tests/integration/cqlengine/columns/test_validation.py

```diff
@@ -18,30 +18,21 @@
     import unittest  # noqa
 
 import sys
-from datetime import datetime, timedelta, date, tzinfo
+from datetime import datetime, timedelta, date, tzinfo, time
 from decimal import Decimal as D
 from uuid import uuid4, uuid1
 
 from cassandra import InvalidRequest
-from cassandra.cqlengine.columns import TimeUUID
-from cassandra.cqlengine.columns import Ascii
-from cassandra.cqlengine.columns import Text
-from cassandra.cqlengine.columns import Integer
-from cassandra.cqlengine.columns import BigInt
-from cassandra.cqlengine.columns import VarInt
-from cassandra.cqlengine.columns import DateTime
-from cassandra.cqlengine.columns import Date
-from cassandra.cqlengine.columns import UUID
-from cassandra.cqlengine.columns import Boolean
-from cassandra.cqlengine.columns import Decimal
-from cassandra.cqlengine.columns import Inet
+from cassandra.cqlengine.columns import TimeUUID, Ascii, Text, Integer, BigInt, VarInt, DateTime, Date, UUID, Boolean, \
+    Decimal, Inet, Time, UserDefinedType, Map, List, Set, Tuple, Double, Float
```

Comment:
Yeah, we should standardize on something in the 80-120 range, but figuring out a style guide hasn't been a priority. Again, this is my personal preference, so if you have strong feelings, go with it.